README.md

```diff
@@ -27,9 +30,9 @@ import requests
 # This data defines a binary cloudevent
 attributes = {
     "Content-Type": "application/json",
```

Comment:
I think this should be `datacontenttype`?

Reply:
Is this needed in the first place?